.github/workflows/build.yml

```diff
@@ -0,0 +1,22 @@
+name: Java CI
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Java + Loom early access build
+    steps:
+      - name: Checkout source code
+        uses: actions/checkout@v3
+      - name: setup
+        uses: oracle-actions/setup-java@v1
+        with:
+          website: jdk.java.net
+          release: Loom
```

Comment:
Can't we use directly Java 19 EAP?

Reply:
Yes, I will find the right configuration for Java 19 ea